MaterialShowcase/MaterialShowcase.swift

```diff
@@ -52,6 +52,10 @@ public class MaterialShowcase: UIView {
   // Background
   public var backgroundPromptColor: UIColor!
   public var backgroundPromptColorAlpha: CGFloat!
+  // Settings for tap zone
+  //isTapRecognizerForTagretView == false - recognize tap for all Mathetial view
+  //isTapRecognizerForTagretView == true  - recognize tap for only targetView rect
```

Comment:
Can you update documentation?
- Fix spelling "Mathetial view" -> "MaterialView"
- Update it to be more concise:
i.e: 
// Tap zone settings
// - false: recognize tap from all displayed MaterialView
// - true: recognize tap for targetView area only